src/clj/puppetlabs/services/analytics/dropsonde.clj

```diff
@@ -0,0 +1,18 @@
+(ns puppetlabs.services.analytics.dropsonde
+  (:require [clojure.java.shell :refer [sh]]
+            [clojure.tools.logging :as log]
+            [puppetlabs.i18n.core :as i18n]))
+
+(def puppet-agent-ruby "/opt/puppetlabs/puppet/bin/ruby")
+(def dropsonde-dir "/opt/puppetlabs/server/data/puppetserver/dropsonde")
+(def dropsonde-bin (str dropsonde-dir "/bin/dropsonde"))
+
+(defn run-dropsonde
+  []
+  (let [result (sh puppet-agent-ruby dropsonde-bin "submit"
```

Comment:
Right now, this doesn't quite work, because when we go to initialize Puppet's settings, we're running as `puppet`, not root, which means we're looking in the wrong spot for things like `confdir` and `codedir`.